src/index.js

```diff
@@ -93,18 +109,29 @@ const chooseWeighted = (options) => {
   }
 };
 
+/**
+ * @param {Object} drop An item to be added to the player's inventory
+ * @param {string} drop.mat The item's name
+ */
+const addItem = (drop) => {
+  let result = inventory.find(loot => loot.mat === drop.mat);
+  if (result) {
+    result.quantity += 1;
+  } else {
+    inventory.push({ mat: drop.mat, quantity: 1 });
+  }
+};
+
```

Comment:
Hey @foxfriends, did I do the commenting correctly?

Reply:
oho now i see it

Reply:
anyway, only change i would suggest is that capital O in `Object`
```suggestion
/**
 * @param {object} drop An item to be added to the player's inventory
 * @param {string} drop.mat The item's name
 */
const addItem = (drop) => {
  let result = inventory.find(loot => loot.mat === drop.mat);
  if (result) {
    result.quantity += 1;
  } else {
    inventory.push({ mat: drop.mat, quantity: 1 });
  }
};

```